src/aics-image-viewer/components/Toolbar/ViewModeRadioButtons.tsx

```diff
@@ -1,22 +1,22 @@
 import React from "react";
 import { Radio } from "antd";
 
-import viewMode from "../../shared/enums/viewMode";
-const viewModeMapping = viewMode.mainMapping;
+import { ViewMode } from "../../shared/enums";
+// Hack to allow iterating over const enum
+const viewModes = [ViewMode.threeD, ViewMode.xy, ViewMode.xz, ViewMode.yz];
 
-export default function ViewModeRadioButtons(props: { mode: symbol; onViewModeChange(newMode: symbol): void }) {
+export default function ViewModeRadioButtons(props: { mode: ViewMode; onViewModeChange: (newMode: ViewMode) => void }) {
   const onChangeButton = ({ target }) => {
-    let mode = viewMode.STRING_TO_SYMBOL[target.value];
-    if (props.mode !== mode) {
-      props.onViewModeChange(mode);
+    if (props.mode !== target.value) {
+      props.onViewModeChange(target.value);
     }
   };
 
   return (
     <Radio.Group onChange={onChangeButton} value={props.mode.toString()}>
-      {Object.values(viewModeMapping).map((mode, index) => (
+      {viewModes.map((mode, index) => (
```

Comment:
I'd remove the comment that describes this as a hack because you are truly defining the UI display order here with the viewModes array.  (Could rename it to viewModesDisplayOrder or some such...)  or write it like this:
```
{[ViewMode.threeD, ViewMode.xy, ViewMode.xz, ViewMode.yz].map((mode, index) => (
```
but maybe that's slightly too verbose?